sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/aio/_azureappconfigurationproviderasync.py

```diff
@@ -391,7 +420,76 @@ async def refresh(self, **kwargs) -> None:
             elif need_refresh and self._on_refresh_success:
                 await self._on_refresh_success()
 
+    def refresh_configuration_settings(self, configuration_settings, sentinel_keys, timer, **kwargs) -> None:
+        success = False
+        need_refresh = False
+        updated_feature_flags = False
+        headers = _get_headers("Watch", uses_key_vault=self._uses_key_vault, **kwargs)
+        try:
+            for (key, label), etag in sentinel_keys.items():
+                etag, update, is_feature_flag, configuration_settings = self._check_configuration_settings(
+                    key, label, etag, headers, configuration_settings, **kwargs
+                )
+                if update and not is_feature_flag:
```

Comment:
Would it be more readable to have an `if else` statement under `if update`
Like below:

```python
if update:
 if is_feature_flag:
    ...
  else:
    ...